app/callbacks/display_callbacks.py

```diff
@@ -209,19 +207,14 @@ def update_image_and_bbox(slider_value, alert_data, alert_list):
     - int: Maximum value for the image slider.
     """
     img_src = ""
-    bbox_style = {}
-    bbox_divs: List[html.Div] = []  # This will contain the bounding box as an html.Div
+    bbox_style = {"display": "none"}  # Default style for the bounding box
```

Comment:
Same spirit as https://github.com/pyronear/pyro-platform/pull/169/files#r1831286489, if you need to temporarily hide an element, you can use this style attribute to do so instead of removing it from the DOM altogether and re-adding it when it needs to be shown again.

Reply:
same answer then :) thanks !